BLSQ/iaso
# SLEEP-1439 Add permission for offline setup zip download

Also: Enable task api in case of access to offline setup file.

Without this, the user will see an error when the frontend tries to render the progress bar.

Related JIRA tickets : [SLEEP-1439](https://bluesquare.atlassian.net/browse/SLEEP-1439) (already in Trypelim)

## Print screen / video

![2024-08-21_10-27](https://github.com/user-attachments/assets/997ee462-08c5-4770-a0ea-7744843089c7)

## How to test

- There should be a new permission "Télécharger l'installation hors ligne de l'application mobile" visible in the user management screen
- When you have this new permission, you see this icon appear as a new user action, allowing you to download a mobile app setup-zip as this user:
![2024-08-21_14-35](https://github.com/user-attachments/assets/edea4a35-22a2-491b-a185-0b70cd6880e0)
- If you don't have the permission, you don't see the action on the users

[SLEEP-1439]: https://bluesquare.atlassian.net/browse/SLEEP-1439?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
